### PR TITLE
Added blog post

### PR DESCRIPTION
**Pull request by Juanvulcano**
Hi there ! 
I added my Code-In experience to the site and changed the url from my game that was mistakenly changed after they moved it from the gallery :)
Thanks to all the mentors for teaching us, it have been a great experience !  
Please take a look 
http://juanvulcano.github.io/post/goodbye-gci/

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Juanvulcano.github.io/gci15.fossasia.org

**Comment by jigyasa-grover** [comment]
Thanks for the suggestions @Juanvulcano .

The image attributes are as:

```
img[Attributes Style] {
    width: 743px;
    height: 549px;
}
```

whereas we require images of size 240*240 (Issue #671), could you please rectify it ?

**Comment by Juanvulcano** [comment]
Hi Jigyasa ! 
Sure, just updated the size :)

**Comment by jig08** [comment]
Awesome !

**Comment by jigyasa-grover** [comment]
Hey there @Juanvulcano 
screenshot from 2016-01-25 14 12 36

Why is your entry extending beyond the desired section ?

**Comment by Juanvulcano** [comment]
@jig08 This is quite curious actually, I'm working into this but can't seem to find a way to fix this. I'll try changing the image or maybe it have something to do with the comma in the title

**Comment by jigyasa-grover** [comment]
Okay, find your way .. :wink:

File: index.html [hunk 1]
```diff
@@ -591,6 +591,22 @@ <h1 class="section-h1"><span class="white-border">Student Games</span></h1>
           <div class="container">
             <section>
               <div class="row">
+		<div class="col-xs-4 col-sm-4 col-md-3 col-lg-2">
+                  <div class="card">
+                    <a href="https://github.com/Juanvulcano/GCI/tree/master/Fossasia/Week1/Treasurehunt">
+                      <div class="avatar img-circle">
+                        <img class="card-img-top" src="img/gallery/Treasurehunt.png" data="img/gallery/Treasurehunt.png" alt="Juanvulc$
```
Comment (yasoob) [inline index.html:598]: @Juanvulcano @jig08 Why is there a Dollar sign `$` here instead of a closing tag? :wink:

**Comment by mananwason** [comment]
@yasoob This has been fixed in #681

File: index.html (hunk 1 above)
Comment (mananwason) [inline index.html:598]: @yasoob This has been fixed in #681

Comment (yasoob) [inline index.html:598]: @mananwason Ah! Didn't see that PR :smile:

Comment (mananwason) [inline index.html:598]: No issues !! :smile:

**Pull request merged**